docs/DeveloperGuide.md

```diff
@@ -322,15 +323,93 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case: Sort contacts**
+
+**Actor:** User
+
+**MSS**
+
+1.  User chooses what field to sort by (e.g. name, specialization, graduation year) and which order to sort by (ascending or descending).
+
+1.  System shows list of user’s contacts, sorted by the specified field and in the specified order.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The user has no contacts to sort.
+
+    * 1a1. System shows an empty contact list.
+
+      Use case ends.
+
+* 1a. The specified field is missing.
+
+    * 1a1. System shows an error message.
+
+      Use case ends.
+
+* 1a. The specified field is invalid.
+
+    * 1a1. System shows an error message.
+
+      Use case ends.
+
+ * 1a. The specified sorting order is invalid.
+
+    * 1a1. System shows an error message.
+
+      Use case ends.
+
+ * 1a. Certain contacts do not have any data in the specified field (e.g. no email address stored)
+
+    * 1a1. System shows list of user’s contacts, sorted by the specified field and in the specified order. Any contacts without that field specified are put at the end of the list.
+
+      Use case ends.
+
+
+
+
+**Use case: Search contacts by name**
+
+**Actor:** User
+
+**MSS**
+
+1.  User specifies the text they would like to search.
+
+1.  System shows list of user’s contacts with names containing the searched text.
```

Comment:
Should be 2.

Reply:
Maybe can take the chance to double-check especially for use cases!

Reply:
Hi, the use of all 1s is intentional here as it will be converted into the correct numbering when viewed. It is also recommended in the [Markdown style guide](https://se-education.org/guides/conventions/markdown.html) for longer lists. However, I realize that it may make more sense to type out the numbering for use cases specifically, as the specific numbers are referenced in the extensions. I'll make the changes here since it makes sense.